File: javascript-closure-angular-client/API/Client/ValidationError.js
goog.provide('API.Client.ValidationError');

/**
 * @record
 */
API.Client.ValidationError = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.ValidationError.prototype.message;

/**
 * @type {!Array<!API.Client.Error>}
 * @export
 */
API.Client.ValidationError.prototype.errors;

File: dot-net-client/src/Avalara.AvaTax.Brazil.RestClient/Model/ValidationError.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Avalara.AvaTax.Brazil.RestClient.Model
{
    /// <summary>
    /// ValidationError
    /// </summary>
    [DataContract]
    public class ValidationError
    {
        /// <summary>
        /// Gets or Sets Message
        /// </summary>
        [DataMember(Name = "message", EmitDefaultValue = false)]
        public string Message { get; set; }

        /// <summary>
        /// Gets or Sets Errors
        /// </summary>
        [DataMember(Name = "errors", EmitDefaultValue = false)]
        public List<Error> Errors { get; set; }
    }
}
